src/core/hooks/useKeyBackupReminder.tsx

```diff
@@ -1,70 +1,62 @@
-import { Button } from "../../components/UI/Button.tsx";
-import type { CookieAttributes } from "js-cookie";
+import { Button } from "@components/UI/Button.tsx";
 import { useCallback, useEffect, useRef } from "react";
-import useCookie from "./useCookie.ts";
-import { useToast } from "./useToast.ts";
+import { useToast } from "@core/hooks/useToast.ts";
+import useLocalStorage from "@core/hooks/useLocalStorage.ts";
 
 interface UseBackupReminderOptions {
   reminderInDays?: number;
   message: string;
   onAccept?: () => void | Promise<void>;
   enabled: boolean;
-  cookieOptions?: CookieAttributes;
 }
 
 interface ReminderState {
   suppressed: boolean;
   lastShown: string;
 }
 
-const TOAST_APPEAR_DELAY = 10_000; // 10 seconds;
-const TOAST_DURATION = 30_000; // 30 seconds;:
-
-// remind user in 1 year to backup keys again, if they accept the reminder;
+const TOAST_APPEAR_DELAY = 10_000; // 10 seconds
+const TOAST_DURATION = 30_000; // 30 seconds
 const ON_ACCEPT_REMINDER_DAYS = 365;
+const STORAGE_KEY = "key_backup_reminder";
 
 function isReminderExpired(lastShown: string): boolean {
   const lastShownDate = new Date(lastShown);
   const now = new Date();
-  const daysSinceLastShown = (now.getTime() - lastShownDate.getTime()) /
-    (1000 * 60 * 60 * 24);
+  const daysSinceLastShown = (now.getTime() - lastShownDate.getTime()) / (1000 * 60 * 60 * 24);
   return daysSinceLastShown >= 7;
 }
 
 export function useBackupReminder({
   reminderInDays = 7,
   enabled,
   message,
-  onAccept = () => {},
-  cookieOptions,
+  onAccept = () => { },
 }: UseBackupReminderOptions) {
   const { toast } = useToast();
   const toastShownRef = useRef(false);
-  const { value: reminderCookie, setCookie } = useCookie<ReminderState>(
-    "key_backup_reminder",
+  const [reminderState, setReminderState] = useLocalStorage<ReminderState | null>(
+    STORAGE_KEY,
+    null
   );
 
-  const suppressReminder = useCallback(
-    (days: number) => {
-      const expiryDate = new Date();
-      expiryDate.setDate(expiryDate.getDate() + days);
+  // Suppress reminder for 10 years if not specified
+  const suppressReminder = useCallback((days: number = 3563) => {
+    const expiryDate = new Date();
+    expiryDate.setDate(expiryDate.getDate() + days);
 
-      setCookie(
-        {
-          suppressed: true,
-          lastShown: new Date().toISOString(),
-        },
-        { ...cookieOptions, expires: expiryDate },
-      );
-    },
-    [setCookie, cookieOptions],
-  );
+    setReminderState({
+      suppressed: true,
+      lastShown: new Date().toISOString(),
```

Comment:
did you mean to do this?
```suggestion
      lastShown: new Date().toISOString(),
      expires: expiryDate.toISOString()
```

Reply:
Thanks for catching this, I'm not sure why I thought I needed this extra logic, I did a small refactor using expires instead of supressed and lastShown props.